### PR TITLE
Add support for symfony 6

### PR DESCRIPTION
**Pull request by mario-fehr**
**What's in this PR?**

This PR updates the bundle to be compatible with Symfony 6.

### CODE REVIEW

**Comment by alexander-schranz** [comment]
@mario-fehr the return types in the `tests/Application/Kernel.php` need to be added:

```php
public function registerBundles(): iterable
```

```php
public function registerContainerConfiguration(LoaderInterface $loader): void
```

```php
protected function getKernelParameters(): array
```

File: composer.json [hunk 1]
```diff
@@ -109,7 +108,6 @@
         "check-coverage": [
             "@php vendor/bin/code-coverage-checker \"Tests/reports/coverage.php\" \"line\" \"100.00\" \"Content\" \"DependencyInjection\""
         ],
-        "deptract": "vendor/bin/deptrac",
```
Comment (mario-fehr) [inline composer.json:109]: A deptract command already exists in composer file.

File: Content/Infrastructure/Sulu/Search/ContentSearchMetadataProvider.php [hunk 2]
```diff
@@ -341,7 +341,7 @@ private function mapProperty(PropertyMetadata $property, IndexMetadata $metadata
                     // @codeCoverageIgnoreStart
                     $metadata->setImageUrlField($this->getContentField($property));
                     break;
-                // @codeCoverageIgnoreEnd
+                    // @codeCoverageIgnoreEnd
```
Comment (mario-fehr) [inline Content/Infrastructure/Sulu/Search/ContentSearchMetadataProvider.php]: Not sure why php-cs-fixer does this.

File: Tests/Unit/Content/Application/ContentWorkflow/ContentWorkflowTest.php [hunk 3]
```diff
@@ -54,14 +54,14 @@ protected function wrapWorkflowMock(ObjectProphecy $workflowMock)
...
             WorkflowInterface {
-            use DimensionContentMockWrapperTrait;
-            use WorkflowMockWrapperTrait;
-
-            public static function getWorkflowName(): string
-            {
-                return 'content_workflow';
-            }
-        };
+                use DimensionContentMockWrapperTrait;
```
Comment (mario-fehr) [inline Tests/Unit/Content/Application/ContentWorkflow/ContentWorkflowTest.php]: Same here

**Comment by mario-fehr** [comment]
@alexander-schranz as discussed in slack upgrade of phpstan packages and linting fixes will be done in an upcoming PR.

**Pull request merged**